package.json

```diff
@@ -1,18 +1,50 @@
 {
-	"private": true,
-	"name": "o-layout",
+	"name": "@financial-times/o-layout",
+	"version": "0.0.0",
+	"description": "Provides page layouts and typography as a starting point to create internal tools or products.",
+	"keywords": [
+		"layout",
+		"documentation",
+		"grid",
+		"page"
+	],
+	"homepage": "https://registry.origami.ft.com/components/o-layout",
+	"bugs": {
+		"url": "https://registry.origami.ft.com/components/o-layout"
+	},
+	"license": "MIT",
+	"type": "module",
+	"browser": "main.js",
+	"scripts": {
+		"start": "npx serve ./demos/local",
+		"build": "npm_config_yes=true npx \"origami-build-tools@11\" install && npm_config_yes=true npx \"origami-build-tools@11\" demo"
+	},
 	"devDependencies": {
-		"eslint": "^7.8.1",
-		"eslint-config-origami-component": "^2.1.0",
-		"origami-ci-tools": "^2.0.2",
+		"eslint": "^7.24.0",
+		"eslint-config-origami-component": "^2.1.1",
 		"remark": "^13.0.0",
-		"remark-lint": "^8.0.0",
-		"remark-preset-lint-origami-component": "^1.1.3",
-		"stylelint": "^13.7.0",
+		"remark-lint": "^7.0.1",
+		"remark-preset-lint-origami-component": "^2.0.0",
+		"stylelint": "^13.12.0",
 		"stylelint-config-origami-component": "^1.0.4"
 	},
-	"scripts": {
-		"start": "npx serve ./demos/local",
-		"build": "npm_config_yes=true npx \"origami-build-tools@^10\" install && npm_config_yes=true npx \"origami-build-tools@^10\" demo"
+	"peerDependencies": {
+		"@financial-times/o-brand": "^4.0.2",
+		"@financial-times/o-colors": "^6.0.1",
+		"@financial-times/o-fonts": "^5.0.0",
+		"@financial-times/o-grid": "^6.0.0",
+		"@financial-times/o-normalise": "^3.0.0",
+		"@financial-times/o-quote": "^5.0.1",
+		"@financial-times/o-spacing": "^3.0.0",
+		"@financial-times/o-typography": "^7.0.1",
+		"@financial-times/o-utils": "^2.0.0",
+		"@financial-times/o-visual-effects": "^3.0.6"
```

Comment:
looks like o-visual-effects didn't release to npm either -- I'll release 4.0.0 now 👍

Reply:
@financial-times/o-visual-effects@4.0.0 is now published onto npm

Reply:
oof. thank you